test/specs/commonTests.js

```diff
@@ -571,7 +572,8 @@ export const implementsShorthandProp = (Component, options = {}) => {
     ShorthandComponent,
     mapValueToProps,
     requiredProps = {},
-    requiredShorthandProps = {},
+    shorthandDefaultProps = {},
+    alwaysPresent,
```

Comment:
Added an `alwaysPresent` flag to shorthand props.  This means it is not tested for "does not exist when null" and other cases.  The `Input` always has a shorthand `input` rendered.